projects/storefrontapp-e2e-cypress/cypress/integration/b2b/my-account/b2b-replenishment-order-details-flow.e2e-spec.ts

```diff
@@ -0,0 +1,32 @@
+import { loginB2bUser } from '../../../helpers/b2b/b2b-checkout';
+import * as replenishmentDetails from '../../../helpers/b2b/b2b-replenishment-order-details';
+import {
+  PLACE_ORDER_REQUEST,
+  POWERTOOLS_BASESITE,
+  USER_REQUEST_CHANNEL,
+} from '../../../sample-data/b2b-checkout';
+
+describe('Replenishment order details', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
+    Cypress.env('USER', USER_REQUEST_CHANNEL);
+    Cypress.env('ORDER', PLACE_ORDER_REQUEST);
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+  });
+
+  afterEach(() => {
```

Comment:
You don't need this afterEach() at the moment since you don't need to restore your local storage between the tests you currently have.